pkg/expression/expression.go

```diff
@@ -1199,6 +1199,17 @@ func scalarExprSupportedByFlash(function *ScalarFunction) bool {
 			tipb.ScalarFuncSig_Reverse:
 			return true
 		}
+	case ast.InetNtoa:
+		sourceType := function.GetArgs()[0].GetType()
+		switch sourceType.GetType() {
+		case mysql.TypeLong, mysql.TypeShort, mysql.TypeTiny:
+			/*
+				TiFlash only support unsigned integer(size <= 4 bytes)
+				MySQL will return default value and warning if failed to cast into 4 bytes unsigned integer(include `Out of range`).
```

Comment:
Why can't let TiFlash following this behavior? I'm afraid for most of the cases, the source type will be mysql.TypeLongLong